openpower/package/sbe-validation/Config.in

```diff
@@ -0,0 +1,32 @@
+config BR2_PACKAGE_SBE_VALIDATION
+        bool "sbe-validation"
+        default y if (BR2_OPENPOWER_PLATFORM && BR2_OPENPOWER_POWER9)
+        select BR2_CPP
+        help
+            Project to build the sbe validation codebase
```

Comment:
What's "the sbe validation codebase" though? This should be a basic explanation of the utility.